tables/he-common-vowels-ihbc.uti

```diff
@@ -0,0 +1,472 @@
+
+# liblouis: he-common-IHBC-vowels
+#
+#-display-name: Hebrew Common Characters-IHBC Vowels
+#-index-name: he-common-characters
+#+language: he
+#+type:literary
+#+contraction:no
+#+system:he-IHBC
+#+dots:6
+
+#-license: lgpl-2.1
+
+# This file is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# This file is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this file; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+# liblouis  comes with ABSOLUTELY NO WARRANTY.
+
+#This is a common character file that supports Hebrew forward  translation. Matthew Yeater, Paul Geoghegan, and Leonard De Ruijter
```

Comment:
@yeatersink Can you mention "IHBC" and "vowels" in this description? Note that I will remove the metadata section at the top because this an include table, not a top-level table.

Reply:
This line is indeed causing a compilation error.